src/components/Pages/FormInput.tsx

```diff
@@ -12,12 +12,40 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
-import { useForm } from 'react-hook-form';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+
+const schema = yup.object({
+	firstName: yup.string().required('First name is required!'),
+	lastName: yup.string().required('last name is required!'),
+	email: yup.string().email().required('Should be valid email address!'),
+	password: yup
+		.string()
+		.min(8, 'Password must be 8 characters long')
+		.matches(/[0-9]/, 'Password requires a number')
+		.matches(/[a-z]/, 'Password requires a lowercase letter')
+		.matches(/[A-Z]/, 'Password requires an uppercase letter')
+		.matches(/[^\w]/, 'Password requires a symbol'),
+});
```

Comment:
extract to separate file called validations (in the same directory as component)